applications/GeoMechanicsApplication/custom_elements/geo_truss_element_base.cpp

```diff
@@ -206,34 +207,36 @@ void GeoTrussElementBase<TDim,TNumNodes>::
 
 //----------------------------------------------------------------------------------------
 template< unsigned int TDim, unsigned int TNumNodes >
-void GeoTrussElementBase<TDim,TNumNodes>::
+ void GeoTrussElementBase<TDim,TNumNodes>::
     CalculateConsistentMassMatrix(MatrixType& rMassMatrix,
                                   const ProcessInfo& rCurrentProcessInfo) const
 {
-    KRATOS_TRY;
+    KRATOS_TRY
+
+    static constexpr unsigned int NDof = TDim * TNumNodes;
```

Comment:
Isn't this unused?

Reply:
It is used in the same function.

Reply:
For the future we can define these in the header, so we can use them all throughout the class.

Reply:
This variable is only used in this function. That's why I defined it in the function but I can also move it to the header file.

Reply:
My point is that there is plenty of places in which the `TDim*TNumNodes` is used. In any case I'd leave this for the future (out of the scope of this PR) :wink: